CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * Enchancement: Support @SentrySpan and @SentryTransaction on classes and interfaces. (#1243)
 * Enchancement: Do not serialize empty collections and maps (#1245)
 * Ref: Simplify RestTemplate instrumentation (#1246)
+* Enchancement: Add overloads for startTrasaction taking op and description (#1244)
```

Comment:
good to be under the new `breaking change` section

Reply:
Fixed.